cloudesire-api-client/src/main/java/com/cloudesire/platform/apiclient/api/UserApi.java

```diff
@@ -175,9 +184,17 @@
     @DELETE( "user/{id}" )
     Call<Void> delete( @Path( "id" ) int id );
 
+    /**
+     * @deprecated look for {@link PaymentMethodApi#delete(int)}
+     */
+    @Deprecated
     @DELETE( "user/{id}/payment" )
     Call<Void> deletePaymentDataForAdmin( @Path( "id" ) int id );
 
+    /**
+     * @deprecated look for {@link PaymentMethodApi#delete(int)}
+     */
+    @Deprecated
     @DELETE( "user/payment" )
     Call<Void> deletePaymentData();
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)